src/driver/amdxdna/aie2_msg_priv.h

```diff
@@ -387,6 +389,64 @@ struct async_event_msg_resp {
 	enum async_event_type	type;
 } __packed;
 
+/***************Start of event tracing data struct ****************/
+
+#define TRACE_EVENT_BUFFER_SIZE	0x2000
+#define MAX_ONE_TIME_LOG_INFO_LEN	16
+
+enum event_trace_destination {
+	EVENT_TRACE_DEST_DEBUG_BUS,
+	EVENT_TRACE_DEST_DRAM,
+	EVENT_TRACE_DEST_COUNT
+};
+
+enum event_trace_timestamp {
+	EVENT_TRACE_TIMESTAMP_FW_CHRONO,
+	EVENT_TRACE_TIMESTAMP_CPU_CCOUNT,
+	EVENT_TRACE_TIMESTAMP_COUNT
+};
+
+struct start_event_trace_req {
+	uint32_t event_trace_categories;
+	enum event_trace_destination event_trace_dest;
+	enum event_trace_timestamp event_trace_timestamp;
+	//DRAM log buffer address and size
+	uint64_t dram_buffer_address;
+	uint32_t dram_buffer_size;
+} __packed;
+
+struct start_event_trace_resp {
+	enum aie2_msg_status status;
+	uint32_t msi_idx;
+	uint64_t current_timestamp;
+} __packed;
+
+struct stop_event_trace_req {
+	uint32_t:32;
```

Comment:
Linux reviewer was asking not use bitfield.
```suggestion
	u32		place_holder;
```

Reply:
updated.